packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -279,6 +287,14 @@ enum VideoEventType {
   /// The video stopped to buffer.
   bufferingEnd,
 
+  /// Represents an event type emitted by the platform implementation when the
```

Comment:
nit: This should follow

https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary and https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph